### PR TITLE
feat(examples): re-added gatsbygram avatar

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Description

Re-added the avatar on the main gatsbygram page.

The avatar was removed due to issues in #12133 - but as of #12371 avatars are working again, I re-added the main page avatar.

### CODE REVIEW

File: examples/gatsbygram/src/components/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        [`@media (min-width: 460px)`]: {
+          width: rhythm(3),
+          height: rhythm(3),
+        },
+        [`@media (min-width: 525px)`]: {
+          width: rhythm(4),
+          height: rhythm(4),
+        },
+        [`@media (min-width: 600px)`]: {
+          width: rhythm(6),
```
Comment (wardpeet) [inline examples/gatsbygram/src/components/Avatar.js]: we've used the value `inherit` before, what's the reason for using `rhythm(6)`?

Comment (FrederikBolding) [reply examples/gatsbygram/src/components/Avatar.js]: As the avatar for some reason is very low resolution, using 'inherit' doesn't look right, instead I chose 'rhythm(6)'.

I can change it back if you like it better the other way.

Comment (FrederikBolding) [reply examples/gatsbygram/src/components/Avatar.js]: Yeah, we could just remove it - it's simpler that way

Comment (FrederikBolding) [reply examples/gatsbygram/src/components/Avatar.js]: @wardpeet Just tested it - The image is way to small without the media query on my screen.

Comment (wardpeet) [reply examples/gatsbygram/src/components/Avatar.js]: mind showing a screenshot?

Comment (FrederikBolding) [reply examples/gatsbygram/src/components/Avatar.js]: image

image

First one is with my changes, last one is without media query

Comment (wardpeet) [reply examples/gatsbygram/src/components/Avatar.js]: oh weird, well just go with yours, it's no big deal anyway 😄

**Pull request merged**